docs/sources/installation/helm/_index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Install Grafana Loki with Helm
+menuTitle: Concept
```

Comment:
https://raw.githubusercontent.com/grafana/mimir/main/docs/sources/operators-guide/deploy-grafana-mimir/_index.md uses `Deploy on Kubernetes`.
```suggestion
menuTitle: Concept
```
```suggestion
menuTitle: Deploy on Kubernetes
```

Reply:
Not sure if my suggestions fits here or not, but `Concept` is a meta point. For the menu title, use a concise version of the title, if feasible.